scripts/terminalserver/terminalserver.jl

```diff
@@ -1,6 +1,6 @@
 # this script basially only handles `Base.ARGS`
 
-Base.push!(LOAD_PATH, @__DIR__)
+Base.push!(LOAD_PATH, joinpath(@__DIR__, "..", "packages"))
```

Comment:
how about locating this file under `scripts` as we did for `error_handler.jl` ? I don't think we will add more files to `terminalserver` directory.

Reply:
I think the only reason I put error_handler.jl in the root is because it is used by multiple things. I kind of like to have things in sub folders per functionality, seems a bit cleaner? If anything, I'd almost want to move `error_handler.jl` into some `shared` folder or something like that.

Reply:
okay, understand.